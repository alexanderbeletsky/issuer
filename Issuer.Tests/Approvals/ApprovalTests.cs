using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ApprovalTests.Asp;
using ApprovalTests.Asp.Mvc;
using ApprovalTests.Reporters;
using Issuer.Controllers;
using Issuer.Repositories;
using NUnit.Framework;

namespace Issuer.Tests.Approvals
{
    [UseReporter(typeof(FileLauncherReporter))]
    public class ApprovalTests
    {
        [Test]
        public void all_issues_test()
        {
            AspApprovals.VerifyUrl("http://localhost:51989/issues/all");
        }

        [Test]
        public void by_id_test()
        {
            AspApprovals.VerifyUrl("http://localhost:51989/issues/show/issues-1");
        }
    }
}
