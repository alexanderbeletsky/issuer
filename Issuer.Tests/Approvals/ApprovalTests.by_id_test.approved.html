<!DOCTYPE html>

<html>
<head><base href="http://localhost:51989/issues/show">
    <meta name="viewport" content="width=device-width" />
    <title>Issuer | Issue</title>
    
    <script src="/Scripts/jquery-1.4.4.js"></script>
<script src="/Scripts/jquery.validate.js"></script>
<script src="/Scripts/jquery.validate.unobtrusive.js"></script>

    <link href="/Content/bootstrap.css" rel="stylesheet"/>
<link href="/Content/bootstrap-responsive.css" rel="stylesheet"/>
<link href="/Content/style.css" rel="stylesheet"/>

    
</head>
<body>
    <div class="navbar navbar-inverse navbar-fixed-top">
        <div class="navbar-inner">
            <div class="container">
            <a href="/" class="brand">Issuer</a>
                <div class="nav-collapse collapse pull-right">
                    <ul class="nav">
                        <li><a href="">Faq</a></li>
                        <li><a href="">Account</a></li>
                        <li><a href="">Support</a></li>
                        <li><a href="">Blog</a></li>
                        <li><a href="">Sign out</a></li>
                    </ul>
                </div>
            </div>
        </div>
    </div>
    
    <div class="container">
        <div id="app" class="row">
            

<legend>Issue: 1 <small class="pull-right">created at: 11/15/2012 12:08:16 PM</small></legend>
<div class="control-group">
    <label for="Headline">Headline:</label>
    12345
</div>
<div class="control-group">
    <label for="Description">Issue description:</label>
    fdsfadsf
</div>
<div class="control-group pull-right">
    <a class="btn btn-info" href="/issues/complete/issues-1">Complete</a>
</div>
        </div>
        <hr/>
        <footer>
            Â© Sitecore 2012
            
        </footer>
    </div>
    
</body>
</html>
